.github/workflows/ci.yaml

```diff
@@ -1,6 +1,12 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
 name: Tests
+
 on:
   pull_request:
+  push:
+    branches:
+      - main
```

Comment:
i remember these tests intentionally not being run on push to the main branch. this is because the tests are run [here](https://github.com/canonical/mysql-operator/blob/main/.github/workflows/release.yaml#L23-L26) in `release.yaml`